UI/BindingSources/XAML/Binding/MainPage.xaml

```diff
@@ -0,0 +1,46 @@
+﻿<Page x:Class="Binding.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Binding"
+	  xmlns:um="using:Uno.Material"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  x:Name="page"
+	  Background="{ThemeResource BackgroundBrush}">
+
+	<Page.DataContext>
+		<local:MainViewModel />
+	</Page.DataContext>
+
+	<StackPanel HorizontalAlignment="Center"
+				VerticalAlignment="Center"
+				utu:SafeArea.Insets="VisibleBounds">
+		<ListView ItemsSource="{Binding Items}">
+			<ListView.ItemTemplate>
+				<DataTemplate>
+					<StackPanel Orientation="Horizontal" Spacing="8">
+						<TextBlock VerticalAlignment="Center" Text="{Binding Text}" />
+						<Button Command="{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}"
+								CommandParameter="{Binding}"
+								Content="Delete" />
+
+						<!--  RelativeSource as TemplatedParent works on Windows but not on Uno  -->
+						<!--  RelativeSource as Self works on Uno but not on Windows  -->
+						<Button Command="{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}"
+								CommandParameter="{Binding DataContext, RelativeSource={RelativeSource Mode=Self}}"
+								Content="Delete" />
```

Comment:
Same as C# Markup here. (https://github.com/unoplatform/Uno.Samples/pull/681#discussion_r1596963271)

Reply:
Resolving as it's the same case as https://github.com/unoplatform/Uno.Samples/pull/681#discussion_r1596963271